sbnanaobj/StandardRecord/SRCRUMBSResult.cxx

```diff
@@ -0,0 +1,32 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRCRUMBSResult.cxx
+////////////////////////////////////////////////////////////////////////
+
+#include "sbnanaobj/StandardRecord/SRCRUMBSResult.h"
+
+namespace caf {
+SRCRUMBSResult::SRCRUMBSResult()
+  : score(-5.f)
+  , tpc_CRFracHitsInLongestTrack(-5.f)
+  , tpc_CRLongestTrackDeflection(-5.f)
+  , tpc_CRLongestTrackDirY(-5.f)
+  , tpc_CRNHitsMax(-5.f)
+  , tpc_NuEigenRatioInSphere(-5.f)
+  , tpc_NuNFinalStatePfos(-5.f)
+  , tpc_NuNHitsTotal(-5.f)
+  , tpc_NuNSpacePointsInSphere(-5.f)
+  , tpc_NuVertexY(-5.f)
+  , tpc_NuWeightedDirZ(-5.f)
+  , tpc_StoppingChi2CosmicRatio(-5.f)
+  , pds_FMTotalScore(-5.f)
+  , pds_FMPE(-5.f)
+  , pds_FMTime(-5.f)
+  , crt_TrackScore(-5.f)
+  , crt_HitScore(-5.f)
+  , crt_TrackTime(-5.f)
+  , crt_HitTime(-5.f)
```

Comment:
These should all be NaNs and the like. The convention is NaNs where the value is literally uninitialized (indicating a bug in the CAFMaker code). -5 is for the case where we have purposely considered the value and there is nothing appropriate to put.

Reply:
Okiedokie, done! I've used `std::numeric_limits<int>::max()` for the ints.